test/draw/RasterRgbTest.ooc

```diff
@@ -84,27 +84,31 @@ RasterRgbTest: class extends Fixture {
 			image := RasterRgb open(this sourceSpace)
 			image2 := image copy()
 
-			image _coordinateSystem = CoordinateSystem YUpward
+			image referenceCount decrease()
+			image = RasterRgb open(this sourceSpace, CoordinateSystem YUpward)
```

Comment:
```ooc
image2 save(file) . free()
image2 = RasterRgb open(file, coordinateSystem)
```
instead of 
```ooc
image2 __coordtinateaSystem = coordinateSystem
```